api/autopilot.go

```diff
@@ -54,19 +51,33 @@ type (
 
 	// HostsConfig contains all hosts settings used in the autopilot.
 	HostsConfig struct {
-		AllowRedundantIPs          bool                        `json:"allowRedundantIPs"`
-		MaxDowntimeHours           uint64                      `json:"maxDowntimeHours"`
-		MinProtocolVersion         string                      `json:"minProtocolVersion"`
-		MaxConsecutiveScanFailures uint64                      `json:"maxConsecutiveScanFailures"`
-		ScoreOverrides             map[types.PublicKey]float64 `json:"scoreOverrides"`
+		AllowRedundantIPs          bool   `json:"allowRedundantIPs"`
```

Comment:
Same thought here: these are values that, for the most part, we don't want users to change without fully understanding them.

Reply:
I agree on that one. I don't think we ever touched it.